packages/theme/package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "@trezor/theme",
```

Comment:
this theme package looks perfectly clear 👍  could you maybe just add some really basic README that could mention, that it is supposed to be used in the suite-native only for now?

Reply:
added